app/views/shared/_our_services.html.slim

```diff
@@ -0,0 +1,16 @@
+.container.mt-5
+  h4.title.text-center OUR SERVICES
+  .service-cards.mt-5
+    .row.justify-content-center
+      .service-card.col-12.col-md-4.text-center.mb-5
+        .image.bg-blue.mb-3
+        p.button.button-primary.my-3 For Artists
```

Comment:
Wait, these buttons don't open their respective pop-ups!

Reply:
@martindefruit 
Send me the design of the popups and I'll make em

Reply:
Thanks :) Yeah, we should talk about what needs to be done after our meeting with them 👍